src/lib/utils.ts

```diff
@@ -47,6 +51,121 @@ export function getMsgIdFromLink(link: string): string {
 	return msgId;
 }
 
+export async function modifyRoleDD(interaction: CommandInteraction, role: Role, isCourse: boolean, dropdownAction: 'ADD' | 'REMOVE'): Promise<boolean> {
+	let rolesMsg: Message;
+	const channel = await interaction.guild.channels.fetch(CHANNELS.ROLE_SELECT) as TextChannel;
+	if (!channel || channel.type !== 'GUILD_TEXT') {
+		const responseEmbed = new MessageEmbed()
```

Comment:
I've made a function in my branch (at least) that generates the error embed automatically. An example of use includes `interaction.reply(generateErrorEmbed('oops'));` Might be worth changing these argument error embeds to that to condense the code

Reply:
Probably a low priority thing though that I/we can work on in our down time during the spring

Reply:
Personally I think that can probably be something that gets done after all the slash commands are merged into (at least) dev, if not main.